about.html

```diff
@@ -0,0 +1,144 @@
+            </a>
+            <a href="#">
+                <i class="fa-brands fa-twitter"></i>
+            </a>
+            <a href="#">English</a>
+            <a href="#">My Page</a>
+            <a href="#">Logout</a>
+        </div>
+        <nav id="navigation">
+            <a id="page-logo" href="./index.html">
+                <img src="images/iCON.png" alt="Logo">
+            </a>
+            <ul id="desktop-nav">
+                <li>
+                    <a href="./index.html" class="nav-list">Home</a>
+                </li>
+                <li>
+                    <a href="#" class="nav-list">Program</a>
+                </li>
+                <li>
+                    <a href="./about.html" id="active">About</a>
+                </li>
+                <li>
+                    <a href="#" class="nav-list">Sponsor</a>
+                </li>
+                <li>
+                    <a href="#" class="nav-list">News</a>
+                </li>
+                <button type="button" id="nav-btn">Join Now</button>
+            </ul>
+        </nav>
+    </header>
+    <div id="hamburger">
+        <a href="#" id="close">
+            <i id="close-icon" class="material-icons">close</i>
+        </a>
+        <ul id="hamburger-ul">
+            <li>
+                <a class="link" href="./index.html"> Home</a>
+            </li>
+            <li>
+                <a class="link" href="#"> Program</a>
+            </li>
+            <li>
+                <a class="link" href="./about.html"> About</a>
+            </li>
+            <li>
+                <a class="link" href="#"> Sponsor</a>
+            </li>
+            <li>
+                <a class="link" href="#"> News</a>
+            </li>
+        </ul>
+    </div>
+    <main>
+        <section id="about-intro">
+            <h3>"Hello! Data Scientists"</h3>
+            <h1>AI & BIG DATA FOR A SMARTER FUTURE EXHIBITION 2023</h1>
+            <P>
+                AI & Big Data Exhibition brings together the community of experts,
+                academics, and activists who showcase the next generation enterprise 
+                technologies and strategies from the world of Data Science and AI for 
+                a smarter future.
+            </P>
+            <div id="contact">
+                <p>Please contact us per Email for any future questions about AI & Big Data Exhibition 2023!</p>
+                <a href="#">ai-bigdata2023@rdnairobi.org</a>
+            </div>
+        </section>
+        <section id="about-logo-section">
+            <div class="logo-exhibition">
+                <h3>AI & Big Data for a Smarter Future Exhibition 2023 Logo</h3>
+                <hr>
+                <p>The logo for AI & Big Data Exhibition was decided through the logo competition from 8. June 2022 to 7. July 2022.</p>
+            </div>
+            <div class="logo">
+                <img src="images/iCON.png" alt="Logo Icon" />
+            </div>
```

Comment:
- [ ] Please make sure that image is clear in order to make your design more beautiful and make it similar to figma template :smile: